apps/ledger-live-mobile/src/components/FabActions/hooks/useAssetActions.tsx

```diff
@@ -74,6 +74,12 @@ export default function useAssetActions({
     return [onRampProviders.length > 0, offRampProviders.length > 0];
   }, [rampCatalog.value, currency]);
 
+  const canSend = useMemo(() => currency?.id !== "avalanchepchain", [currency]);
```

Comment:
@haammar-ledger This is coin-specific logic in a file that should only contain generic code. How do you suggest I re-factor this so that the `send` and `receive` actions buttons are disabled from the asset page?

Reply:
I think it's a first time that a crypto doesn't have send and receive.
Let's keep it like this, meanwhile we will see if we can make a better solution.
Because we want to avoid to have specific string id here I guess
Also I'm not sure but did we have this logic on desktop ?  + It wont block user to access to send / receive from another flow because I think avalachepchain will be on cryptoassets list

Reply:
We got around this on desktop by declaring our own send and receive actions inside `accountActions.jsx`.

https://github.com/LedgerHQ/ledger-live/blob/cdb1c6d8aeb51d2393a9a42d454047c8a448ee60/apps/ledger-live-desktop/src/renderer/families/avalanchepchain/accountActions.jsx#L3